docs/eventing/sink/kafka-sink.md

```diff
@@ -0,0 +1,162 @@
+---
+title: "Apache Kafka Sink"
+weight: 30
+type: "docs"
+---
+
+## Prerequisites
+
+[Knative Eventing installation](./../../install/any-kubernetes-cluster.md#installing-the-eventing-component).
+
+## Installation
+
+1. Install the Kafka controller by entering the following command:
+
+    ```bash
+    kubectl apply --filename {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-controller.yaml" >}}
+    ```
+
+1. Install the Kafka Sink data plane by entering the following command:
+
+    ```bash
+    kubectl apply --filename {{< artifact org="knative-sandbox" repo="eventing-kafka-broker" file="eventing-kafka-sink.yaml" >}}
+    ```
+
+1. Verify that `kafka-controller` and `kafka-sink-receiver` are running,
+by entering the following command:
+
+    ```bash
+    kubectl get deployments.apps -n knative-eventing
+    ```
+
+    Example output:
+
+    ```bash
+    NAME                           READY   UP-TO-DATE   AVAILABLE   AGE
+    eventing-controller            1/1     1            1           10s
+    eventing-webhook               1/1     1            1           9s
+    kafka-controller               1/1     1            1           3s
+    kafka-sink-receiver            1/1     1            1           5s
+    ```
+   
+## Create a Kafka Sink
+
+A Kafka Sink object looks like this:
+
+```yaml
+apiVersion: eventing.knative.dev/v1alpha1
+kind: KafkaSink
+metadata:
+  name: my-kafka-sink
+  namespace: default
+spec:
+  topic: mytopic
+  bootstrapServers:
+   - my-cluster-kafka-bootstrap.kafka:9092
+```
+
+We can connect any `Source` to the `KafkaSink`, like `PingSource`:
+
+```yaml
+apiVersion: sources.knative.dev/v1beta1
+kind: PingSource
+metadata:
+  name: test-ping-source
+spec:
+  schedule: "*/1 * * * *"
+  jsonData: '{"message": "Hello world!"}'
+  sink:
+    ref:
+      apiVersion: eventing.knative.dev/v1alpha1
+      kind: KafkaSink
+      name: my-kafka-sink
+```
+
+We can connect a `Trigger` to the `KafkaSink`, so that we can filter events, before sending them to Apache Kafka,
+by applying the following `Trigger`:
+
+```yaml
+apiVersion: eventing.knative.dev/v1
+kind: Trigger
+metadata:
+  name: my-service-trigger
+spec:
+  broker: default
+  filter:
+    attributes:
+      type: dev.knative.foo.bar
+      myextension: my-extension-value
+  subscriber:
+    ref:
+      apiVersion: eventing.knative.dev/v1alpha1
+      kind: KafkaSink
+      name: my-kafka-sink
+```
+
+## Kafka Producer configurations
+
+Knative exposes all available Kafka producer configurations that can be modified to suit your workloads.
+
+You can change these configurations by modifying the `config-kafka-sink-data-plane` `ConfigMap` in
+the `knative-eventing` namespace.
```

Comment:
Producer of what, events? Make this clearer

Reply:
In Kafka a Producer is a well-defined thing, maybe Event Producer can be confusing since sources are sort of event producers.

Reply:
I would explain a brief little bit maybe then what a Producer is in that context so that someone can understand what this is without having to go and read all the Kafka docs about it, does that sound reasonable?

Reply:
Yes, thanks, I'll update the PR soon.